Model/Config/Source/Region.php

```diff
@@ -2,14 +2,24 @@
 
 namespace Zip\ZipPayment\Model\Config\Source;
 
-use Zip\ZipPayment\MerchantApi\Lib\Model\CommonUtil;
-
 /**
  * @copyright 2020 Zip Co Limited
  * @link      https://zip.co
  */
 class Region implements \Magento\Framework\Option\ArrayInterface
 {
+    protected $_availbaleCountries = array("au","gb","mx","nz","ca","us","ae","sg","za");
```

Comment:
typo here. should be "availbale" -> available

Reply:
hey @simontruong-zip before only these were supported  'au','nz','gb','za','us','mx'

![image](https://user-images.githubusercontent.com/62676065/141284814-a6fb5baa-7c7f-4210-92ee-7d8b92502a65.png)